app/models/case_log.rb

```diff
@@ -416,11 +408,11 @@ def set_derived_fields!
                          weekly_value(tshortfall)
                        end
     self.nocharge = household_charge&.zero? ? 1 : 0
-    self.underoccupation_benefitcap = 3 if renewal == 1 && year == 2021
+    self.underoccupation_benefitcap = 3 if renewal == 1 && collection_start_year == 2021
```

Comment:
This would be better moved into the `if` below to avoid having the same condition twice. Which I think would also make it obvious that we have a bug there? Effectively we have:

```ruby
if is_renewal?
      self.underoccupation_benefitcap = 3 if collection_start_year == 2021
      self.underoccupation_benefitcap = 2 if collection_start_year == 2021
```

So it'll never be 3?

Reply:
I might need @kosiakkatrina's help to understand what the original intent was.